recipes/_attributes.rb

```diff
@@ -18,14 +18,15 @@
   node.default['postfix']['main']['mailbox_command'] = '/usr/bin/procmail -a "$EXTENSION"'
 end
 
-if node['postfix']['main']['smtpd_use_tls'] == 'yes'
+if node['postfix']['main']['smtpd_use_tls'] == 'yes' or ['may', 'encrypt'].include? node['postfix']['main']['smtpd_tls_security_level']
```

Comment:
use of `or` in ruby is not encouraged, use `||` in preference. Also, please use parentheses around the argument to `include?`.